pinax/templates/templates/pinax/points/one_off_points.html

```diff
@@ -0,0 +1,35 @@
+{% extends "admin/base_site.html" %}
+
+{% load i18n admin_modify adminmedia %}
+
+{% block extrastyle %}{{ block.super }}<link rel="stylesheet" type="text/css" href="{% admin_media_prefix %}css/forms.css" />{% endblock %}
```

Comment:
We don't want to ship static as part of the apps either.  This should be moved into starter project sass.

Reply:
nevermind, this is part of the admin system.  hm...

Reply:
Yeah, these templates are pretty tightly tied to the admin system. Feels a bit odd to put them side by side with the rest of the pinax templates given the differences. If there's a better way to do this I'd love to hear!